src/test/features/claim/routes/total.ts

```diff
@@ -89,6 +90,17 @@ describe('Claim issue: total page', () => {
           .set('Cookie', `${cookieName}=ABC`)
           .expect(res => expect(res).to.be.successful.withText(pageContent))
       })
+
+      it('should throw error when claim value is above £10000 including interest', async () => {
+        draftStoreServiceMock.resolveFind('claim', draftStoreServiceMock.aboveAllowedAmountWithInterest)
+        claimStoreServiceMock.mockCalculateInterestRate(0)
+        claimStoreServiceMock.mockCalculateInterestRate(500)
+
+        await request(app)
+          .get(pagePath)
+          .set('Cookie', `${cookieName}=ABC`)
+          .expect(res => expect(res).to.be.redirect.toLocation(ErrorPaths.amountExceededPage.uri))
```

Comment:
Note this clashes with https://github.com/hmcts/cmc-citizen-frontend/pull/1469

Reply:
yeah I'm expecting to have to change this when that gets merged but my change may get in before that so left it with the currently working fix.